README.rst

```diff
@@ -11,6 +11,10 @@ Single Cell Tools
   :target: https://sctools.readthedocs.io/en/latest/?badge=latest
   :alt: Documentation Status
 
+.. image:: https://img.shields.io/snyk/vulnerabilities/github/HumanCellAtlas/lira/requirements.txt.svg?label=Snyk%20Vulnerabilities&logo=Snyk
```

Comment:
Should this link be to `sctools` instead of `lira`?

Reply:
Oops. Yes. Fixing :)

Reply:
Fixed